src/utils/api/account/lsk.test.js

```diff
@@ -0,0 +1,157 @@
+import http from '../http';
+import ws from '../ws';
+import { getAccount, getAccounts } from './lsk';
+import accounts from '../../../../test/constants/accounts';
+
+jest.mock('../http', () => jest.fn().mockReturnValue([]));
+jest.mock('../ws', () => jest.fn().mockReturnValue([]));
+
+describe('API: LSK Account', () => {
```

Comment:
It would be good to add test cases that check if the optional `baseUrl` is passed to http module.